terraform/eks/container-insights-agent/aoc_fargate_deploy.yml

```diff
@@ -0,0 +1,557 @@
+---
+# create adot-collector service account and role binding
+# update annotations with your cluster role arn
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: adot-collector-service-account
+  namespace: default
+  annotations:
+    eks.amazonaws.com/role-arn: arn:aws:iam::611364707713:role/ServiceAccount-eks-test-aoc-role
+
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: adotcol-admin-role
+rules:
+  - apiGroups: [""]
+    resources:
+      - nodes
+      - nodes/proxy
+      - nodes/metrics
+      - services
+      - endpoints
+      - pods
+      - pods/proxy
+    verbs: ["get", "list", "watch"]
+  - nonResourceURLs: [ "/metrics/cadvisor"]
+    verbs: ["get", "list", "watch"]
+
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: adotcol-admin-role-binding
+subjects:
+  - kind: ServiceAccount
+    name: adot-collector-service-account
+    namespace: default
+roleRef:
+  kind: ClusterRole
+  name: adotcol-admin-role
+  apiGroup: rbac.authorization.k8s.io
+
+# collector configuration section
+# update `ClusterName=my-cluster-name` in the env var OTEL_RESOURCE_ATTRIBUTES
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: adot-collector-config
+  namespace: default
+  labels:
+    app: aws-adot
+    component: adot-collector-config
+data:
+  adot-collector-config: |
+    receivers:
```

Comment:
Are we using a fixed config for each of the tests? Can we use a template and render setup?

Reply:
I can add that in later commits. So we can get the testing in.